src/media/mod.rs

```diff
@@ -247,7 +255,7 @@ pub struct UploadBuilder<'a> {
 }
 
 impl<'a> UploadBuilder<'a> {
-    /// Begins setting up a media upload session, with the given data and media type.
+    /// Creates new instance of upload session, with the given data and media type.
```

Comment:
Previous builders in egg-mode are split on what to use here. `DraftTweet` uses something like your change: "Creates a new `DraftTweet` with the given status text." However, the search builders (whose constructors are separate functions `>_>`) use my original form, like here with `search::search`: "Begin setting up a tweet search with the given query."

Since this is an associated function and not a separate one, i'll take your style, but please phrase it like this:

> Creates a new instance of `UploadBuilder` with the given data and media type.

Reply:
ok.